javascript-forms/form-values.js

```diff
@@ -0,0 +1,13 @@
+const $contactForm = document.querySelector('#contact-form');
+
+function handleSubmit(event) {
+  event.preventDefault();
+  const contactData = {};
+  contactData.name = $contactForm.name.value;
+  contactData.email = $contactForm.email.value;
+  contactData.message = $contactForm.message.value;
+  console.log('contactData: ', contactData);
+  document.querySelector('#contact-form').reset();
```

Comment:
You already queried the DOM for this element on line 1, so you can just do `$contactForm.reset()`. There's no need to query the DOM again!

Reply:
got it, thank you!